ansible/roles/kubernetes_setup/tasks/setup_kubernetes.yml

```diff
@@ -24,7 +15,7 @@
 
   - name: Create command line for kubeadm init
     set_fact:
-      kubeadm_command: "kubeadm init --apiserver-advertise-address={{ hostname }}"
+      kubeadm_command: "kubeadm init --apiserver-advertise-address={{ hostname }} --pod-network-cidr 10.18.0.1/16"
```

Comment:
Is this supposed to be hard-coded to this CIDR?

Reply:
I'm not sure how much it matters.  k8s is looking for addresses to give pods.